public/app/features/dashboard/components/PanelEditor/types.ts

```diff
@@ -34,3 +35,28 @@ export interface Props {
   /* Helpful for IntelliSense */
   data: DataFrame[];
 }
+
+export interface OptionPaneRenderProps {
+  panel: PanelModel;
+  plugin: PanelPlugin;
+  data?: PanelData;
+  dashboard: DashboardModel;
+  onPanelConfigChange: (configKey: string, value: any) => void;
+  onPanelOptionsChanged: (options: any) => void;
```

Comment:
@torkelo maybe `unknown` might be better then? Treat it as an opaque type that needs refining before it can be used, to reduce the temptation to make unsafe assumptions about it.

Reply:
could type as as Record<string, any>

Reply:
`Record<string, unknown>` 😌 

just trying to think about how we can enforce preventing unsafe assumptions about the code that leads to runtime errors.